locales/en/stop-page.json

```diff
@@ -3,6 +3,15 @@
     "stopPageEtagPageBody1": "Since you started using this service, someone else has submitted an update to this director's details.",
     "stopPageEtagPageBody2": "If you still need to submit this update, you'll need to ",
     "stopPageEtagStartServiceAgainLink": "start the service again",
-    "stopPageEtagContactUsLink": "Contact us",
-    "stopPageEtagPageBody3": " if you have any questions."
+    "stopPageContactUsLink": "Contact us",
+    "stopPageContactUsText": " if you have any questions.",
+    "stopPagePre2009CannotUseText": "Directors removed before 1 October 2009 must file on paper instead",
+    "stopPagePre2009Header1": "The date the director was removed is before 1 October 2009.",
+    "stopPagePre2009Form288bLinkText1": "You'll need to file the",
+    "stopPagePre2009Form288bLinkText2": " 288b form 'Terminating appointment as director or secretary' ",
+    "stopPagePre2009Form288bParagraphText": " on paper for directors removed before this date.",
+    "stopPagePre2009WhereToContactUsText1": "If the company is in Northern Ireland, email us at",
+    "stopPagePre2009WhereToContactUsText2": "and ask for form 296 - 'Change of director or secretary or change of particulars'.",
+    "stopPagePre2009RemovedDateTex1": "If the date entered is not correct, you can ",
+    "stopPagePre2009RemovedDateTex2": " enter a different date"
```

Comment:
changes look good, just noticed a small typo in the last 2 keys, it says `Tex` instead of `Text`